web_client/templates/dialogs/editRegionOfInterest.pug

```diff
@@ -0,0 +1,39 @@
+.modal-dialog(role='document')
+  .modal-content
+    form.modal-form(role='form', name='editArea')
+      .modal-header
+        button.close(type='button', data-dismiss='modal', aria-label='Close')
+          span(aria-hidden='true') &times;
+        h4.modal-title Edit Area
+      .modal-body
+        .form-group
+          label(for='h-element-mag') Magnification
+          input#h-element-mag.update-form.input-sm.form-control(
+            type='number', min=1, max=maxMag, step=0.1, value=magnification)
+        .form-group
+          label(for='download-image-format') Format
+          select#download-image-format.update-form.form-control(name='format')
+            option JPEG
+            option PNG
+            option TIFF
+        .form-group
+          label(for='h-element-width') Width
+          input#h-element-width.update-form.form-control(
+            type='text', min=0, step=1, value=scaleWidth, disabled)
+        .form-group
+          label(for='h-element-height') Height
+          input#h-element-height.update-form.form-control(
+            type='text', min=0, step=1, value=scaleHeight, disabled)
+        .form-group
+          label(for='nb-pixel') Number of Pixels
+          input#nb-pixel.update-form.form-control(
+            type='text', value=numberOfPixel, disabled)
+        .form-group
+          label(for='size-file') Approximate size of the file
+          input#size-file.update-form.form-control(type='text', value=fileSize, disabled)
+        .g-validation-failed-message.hidden
+      .modal-footer
+        a#download-area-link.h-download-link.btn.btn-small.btn-primary
```

Comment:
In Girder's ecosystem, we have a convention of prefixing class and id names with a letter.  In girder core, we use `g-`.  In HistomicsTK, it is `h-`.  Could you add `h-` to all of the custom names you defined?

Reply:
Thanks I didn't know !!